packages/libc++/build.sh

```diff
@@ -3,6 +3,7 @@ TERMUX_PKG_DESCRIPTION="C++ Standard Library"
 TERMUX_PKG_LICENSE="NCSA"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=$TERMUX_NDK_VERSION
+TERMUX_PKG_REVISION=1
```

Comment:
You sure this is needed? It wasn't bumped on previous NDK version updates.

Reply:
It has to be rebuild now since it's a major ndk update (and libc++ using packages have to be rebuilt as well I guess), have to let CI know somehow that version has changed or else it won't build it. We could also upload the package manually, or tell CI to build it manually after pushing the ndk-r25 branch, but this way seems easiest

Reply:
OK, since previous NDK version updates didn't require this revision bump, I thought this wasn't needed. Those were built and uploaded manually?

Reply:
Yeah, pretty sure I built and uploaded packages manually for some earlier ndk updates, maybe it will still be easiest to do that manually when bumping all libc++ packages (but for now we can make CI test-build by touching the build recipes)